state/remote/remote_test.go

```diff
@@ -100,6 +106,10 @@ func (c *mockClient) Delete() error {
 	return nil
 }
 
+func (c *mockClient) EnableForcePush() {
```

Comment:
That "Implements ClientForcePusher" would be relevant here

Reply:
Moved down to this line 👍